### ISSUE TITLE
i like this library  can you make it a static library that I can import into my iOS project

### ISSUE THREAD

**Issue by tomubc08** [sergeystoma/tinyvnc#1]

**Comment by sergeystoma** [comment]
Yeah, I'll add a static library when have a moment. Meanwhile, adding it to an iOS project is pretty ease, requires just dropping in a few files as described in "Building for XCode Step by Step" section in readme.
